views/messageboard/results.erb

```diff
@@ -0,0 +1,25 @@
+<html lang="en" class="no-js">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,700|Merriweather' rel='stylesheet' type='text/css'>
+
+	<link rel="stylesheet" href="styles.css"> 
+	
+</head>
+<body>
+  <main>
+    <h1> Here are the results </h1>
+    <br>
+    <br> 
+    <ul>
+      <% @peeps.each do |peep| %>
+        <li>
+            <%=  peep.message  %> 
+            <%=  peep.time  %>
+        </li>
+      <% end %>
```

Comment:
Is there a way you could combine this view with the `views/messageboard/index.erb` view so that you can reuse some of the shared code for displaying peeps? 

Yours is a fair solution and implementing what I suggest above might require some refactoring of your model and controller code to, so I'm only including this here as food for thought.